src/main/java/io/vertx/core/net/NetServerOptions.java

```diff
@@ -400,16 +400,16 @@ public NetServerOptions setSni(boolean sni) {
   /**
    * @return whether the server supports HA Proxy protocol
    */
-  public boolean isProxyProtocolSupport() { return proxyProtocolSupport; }
+  public boolean isUseProxyProtocol() { return useProxyProtocol; }
 
 
   /**
    * Set whether the server supports HA Proxy protocol
    *
    * @return a reference to this, so the API can be used fluently
    */
-  public NetServerOptions setProxyProtocolSupport(boolean proxyProtocolSupport) {
-    this.proxyProtocolSupport = proxyProtocolSupport;
+  public NetServerOptions setUseProxyProtocol(boolean useProxyProtocol) {
```

Comment:
rather than support we should say "Set whether the server uses the HA Proxy protocol" , the "supports" words indicate it can be optional and it is actually not